static/js/breakpoint-handler.js

```diff
@@ -22,18 +24,18 @@ function getViewportDimensions() {
  *  enter: callback when breakpoint region entered,
  *  leave: callback when breakpoint region exited
  * }
- * 
+ *
  */
 
-BreakpointHandler = function (opts) {
-    opts || (opts = {});
+function BreakpointHandler(opts) {
```

Comment:
Do we have any rules on function declarations vs expressions? I've always been in the "write it in the order you need it" mentality, but I can see how taking advantage of hoisting is helpful.